src/components/advanced-color-picker/advanced-color-picker.stories.mdx

```diff
@@ -64,7 +66,6 @@ Shows how the color picker can be used to select a color from a predefined set.
             setOpen(false);
           }}
           onBlur={() => {}}
-          open={open}
```

Comment:
I don't think we should remove this line. I tested the story out and am rather surprised that it seems you can still open and close the picker's dialog, I presume because it's now worked in "uncontrolled" fashion" in which case the whole `open` state variable isn't actually being used.

I know you wanted to change this so it's not open when the page first loads. The correct way to make that change would be to change the initial value in the `useState` call from `true` to `false`.

However both that, and the change you've made here, will mean that Chromatic will stop picking up any visual changes in the picker - it doesn't look like this story has Chromatic disabled (nor should it I don't think). So we should probably have it open by default when Chromatic is running, and not otherwise. We have examples of doing this elsewhere on dialog-like components - using the `isChromatic` util to return a "defaultOpenState` and use that in the `useState` call - if you search the repo for those names you'll find plenty of examples to copy.

Reply:
Yeah I agree, I've utilised the util, but I've had to refactor the stories file type from `.mdx` to `.tsx`, and I've also decided to refactor the test story too. I'll do these as a separate commit, so it's slightly easier to track 👍